packages/rum/test/specs/index.spec.js

```diff
@@ -108,6 +108,7 @@ describe('index', function() {
             },
             reason => {
               fail('Failed sending error:', reason)
+              done()
```

Comment:
this was hanging the test on failures.

Reply:
The docs suggest using `done.fail()`! I couldn't find anything in regards to both calling `fail` and `done`! Just want to make sure calling `done` after `fail` doesn't pass the test inadvertently!

Reply:
The test would still fail in this case, done is just to make sure we dont wait till the timeout and report the error as soon as possible.